### PR TITLE
suggest improvements for cloud formation stack naming scheme

### PR DESCRIPTION
**Pull request by xh3b4sd**
In scope of the Node Pools story we rename a couple of things. So for instance the the CF stacks. Tuomas brought up how the new CF stack should be called and it would be good to have a reasonable naming scheme somehow. Right now we have this. 

```
CPI - Control Plane Initializer
CPF - Control Plane Finalizer
```

```
TCCP - Tenant Cluster Control Plane
TCDP - Tenant Cluster Data Plane
```

How should this be named? In this PR I made a suggestion on which your feedback is required. See also https://github.com/giantswarm/aws-operator/pull/1725#discussion_r300400240.

### CODE REVIEW

**Review by MarcelMue — APPROVED** [review]
Fine with me

**Comment by xh3b4sd** [comment]
More please. I want 3 or 4 approvals on this bad boy.

**Review by kopiczko — APPROVED** [review]

**Comment by xh3b4sd** [comment]
@giantswarm/sig-operator please review and approve when you agree or discuss at least. I would like to have at least two more approvals here.

**Comment by xh3b4sd** [comment]
I think everyone should have read their Emails after 24 hours. When nobody wants to express their opinion further, I go with this now.

**Pull request merged**
